robottelo.properties.sample

```diff
@@ -333,10 +333,24 @@ ssh_key=
 # "/var/lib/libvirt/images/".
 # libvirt_image_dir=/var/lib/libvirt/images
 
-# Section for PostUpgrade Verification Tests
-# [upgrade]
-# Option for providing a preUpgrade yaml data file location on server
-# upgrade_data=
+# Section for Upgrade Scenarios Verification Tests
+[upgrade]
+# RHEVM capsule hostname
+# rhev_cap_host=
+# Custom capsule hostname
+# capsule_hostname=
+# RHEVM capsule activation key
+# rhev_capsule_ak=
+# Custom capsule activation key
+# capsule_ak=
+# The satellite end version
+# to_version=
```

Comment:
```suggestion
# to_version=6.5
```

Reply:
No, We dont want to hardcode as this will set from CI.

Because while upgrading from 6.5 to 6.6, this hardcoded value of 6.5 for TO_VERSION will have complications.

We have it in master though as 6.6 and there is no harm having it there.

Reply:
This was wantedky been removed.

Reply:
Just a query .. Once we cut the branch for 6.6 then this will remain 6.6 and have to update/remove ? Is there anyway if could keep without version in master too ?

Reply:
As  discussed you shall raise another PR to remove from master branch